WorkManagerSample/app/src/androidTest/java/com/example/background/ImageOperationsTest.kt

```diff
@@ -35,10 +35,10 @@ import androidx.work.Configuration
 import androidx.work.WorkManager
 import androidx.work.testing.SynchronousExecutor
 import androidx.work.testing.WorkManagerTestInitHelper
-import com.example.background.Constants.KEY_IMAGE_URI
```

Comment:
keep the packages the same, so we can merge back easily

Reply:
:+1: